### ISSUE TITLE
Alloy role checks for exposed port even if expose_port is false

### ISSUE THREAD

**Issue by rvandam** [grafana/grafana-ansible-collection#449]
The deploy tasks part of the alloy role has a task `Verify that Alloy URL is responding` which checks the accessibility of the local alloy port even if the `alloy_expose_port` variable is false. This causes the task to inevitably fail since the port was never set up in the first place. This task should be skipped if expose_port is false.

**Comment by voidquark** [comment]
Closing as explained here: https://github.com/grafana/grafana-ansible-collection/pull/450
